java/build.gradle

```diff
@@ -7,10 +7,11 @@ buildscript {
         maven { url "https://maven.fullstory.com" }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.3'
-        classpath 'com.fullstory:gradle-plugin-local:1.3.0'
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
+        classpath 'com.android.tools.build:gradle:3.5.3'
+        // always ue the latest FS version
+        classpath 'com.fullstory:gradle-plugin-local:+'
```

Comment:
I'm okay with this for this demo project, but we should probably add a comment that this is NOT recommended

Reply:
added a comment